DataFormats/Detectors/TPC/include/DataFormatsTPC/dEdxInfo.h

```diff
@@ -0,0 +1,47 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file dEdxInfo.h
+/// \author David Rohr
+
+#ifndef DEDXINFO_H
+#define DEDXINFO_H
+
+#include "GPUCommonRtypes.h"
+
+namespace o2
+{
+namespace TPC
+{
+struct dEdxInfo {
+  float dEdxTotIROC;
```

Comment:
should we default assign values here?

Reply:
It is only a struct, and the GPU tracker allocates without constructor some chunk of memory and then casts to the dEdx structure. So default values would not be assigned.
Also, the current OpenCL compiler will not understand C++11 "= 0.f" semantics, and I didn't want to introduce a real constructor.